### ISSUE TITLE
Accessibility: Start B.NJ accessibility review

### ISSUE THREAD

**Issue by fiorella-io** [newjersey/navigator.business.nj.gov#4436]
***DRAFT - DEPENDS ON  https://github.com/newjersey/navigator.business.nj.gov/issues/4435**
____

**Problem Statement:** 

**Benefit Hypothesis:**
- Auditing B.NJ first could be an "easy" way to kick off an audit - less complex than Navigator
- Recommend starting with the most traffic areas first, which is B.NJ

**Success Criteria:** 
- Audit completed for [defined scope]

**In-scope:** audit, analyzing results, sharing findings

**Out-of-scope:** 

**Assumptions:**

**Draft User Stories:**

**Feature Readiness Check List:**
- [ ] Go through the WCAG checklist (maybe with Joe?) to understand which items are applicable to B.NJ
- [ ] Define scope based on Accessibility recommendation convo
- [ ] Conduct audit
- [ ] Analyze findings
- [ ] Present/devlier findings
- [ ] Create tickets for identified next steps

**Comment by amandabeall** [comment]
**Notes for Design Grooming**

Notes
- Dependent on https://github.com/newjersey/navigator.business.nj.gov/issues/4435 
- WCAG 2.2 checklist - https://docs.google.com/spreadsheets/d/1E3bKKG4vI1ucWkUW4HNTMdy9wOeLz9Ti/edit#gid=1254264225

AC
- Define scope during convo with stakeholders
- Use WCAG checklist to conduct an audit - AA standard
- Deliver a 1 pager with audit results
- Present findings with team

**Comment by amandabeall** [comment]
@carye3 per this morning's discussion, do you think I should switch this ticket with #4569? Seem like Navigator is the top priority and now B.NJ

**Comment by carye3** [comment]
@dsullivan-fearless @skeeJay this ticket was dependent on #4435, which was delayed due to scheduling issues. This is being moved to backlog (#4569 is being prioritized above this)
